lib/evmone_precompiles/bls.cpp

```diff
@@ -0,0 +1,57 @@
+#include "bls.hpp"
+#include <blst.hpp>
```

Comment:
```suggestion
#include <blst.h>
```

This also eliminates the need of `blst::`